Library/Formula/rancher-compose.rb

```diff
@@ -0,0 +1,19 @@
+class RancherCompose < Formula
+  homepage "https://github.com/rancher/rancher-compose"
+  url "https://github.com/rancher/rancher-compose/archive/v0.2.6.tar.gz"
+  sha256 "eb52c47ac42b2ecf5cb33d6c78faac8f91b040b23801e42638c4d247db2e1590"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = "#{buildpath}:#{buildpath}/Godeps/_workspace"
+    mkdir_p "#{buildpath}/src/github.com/rancherio"
+    ln_s buildpath, "#{buildpath}/src/github.com/rancherio/rancher-compose"
+    system "go", "build", "-ldflags", "-w -X github.com/rancherio/rancher-compose/version.VERSION #{version}", "-o", "bin/rancher-compose"
+    bin.install "bin/rancher-compose"
+  end
+
+  test do
+    system "bin/rancher-compose", "help"
```

Comment:
Sorry I've copied it from another formula, the problem is, any other command does something on a remote server, like awscli https://github.com/Homebrew/homebrew/blob/6c9b6fe98f0a7723132fc1b69f793b8caccd1f5e/Library/Formula/awscli.rb#L119